base_layer/core/src/chain_storage/postgres_db/migrations/2020-04-07-13-32_block_headers/up.sql

```diff
@@ -0,0 +1,20 @@
+create table block_headers (
+    hash TEXT NOT NULL PRIMARY KEY,
+    height BIGINT NOT NULL,
+    version INT NOT NULL,
+    prev_hash TEXT NOT NULL,
+    TIMESTAMP BIGINT NOT NULL,
```

Comment:
This seems like a mistake

Reply:
how so?
Block timestamps are unix epoch? So its a big int?

Reply:
TIMESTAMP is an SQL keyword

Reply:
thanks, changed to time_stamp

Reply:
Timestamp is fine, but it is uppercase when the other columns are lowercase